src/App.js

```diff
@@ -1,13 +1,58 @@
+import React from 'react'
 import { BrowserRouter } from 'react-router-dom';
 import { Navigation } from './components/shared';
-import { Footer } from './components/mainPage/footer/Footer';
+import { Footer } from './components/mainPage/Footer/Footer';
+import AccountForm from './components/accountForm/accountForm.jsx';
 import Views from './Views';
 
 function App() {
+    const [isLoggedIn, setIsLoggin] = React.useState(false);
+    const [modalShow, setModalShow] = React.useState(false);
+    const [isNewUser, setIsNewUser] = React.useState(false);
+
+    const onSignInHandler = (e) => {
+        setIsNewUser(false);
+        e.preventDefault();
```

Comment:
i think this one should be triggered right after function is called (one line above)